hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/keyvalue/KvMetadataAvroResolverTest.java

```diff
@@ -66,7 +67,7 @@ public void when_noKeyOrThisPrefixInExternalName_then_usesValue(boolean key) {
         KvMetadata metadata = INSTANCE.resolveMetadata(
                 key,
                 List.of(field("field", QueryDataType.INT, "extField")),
-                emptyMap(),
+                Map.of("schema.registry.url", "http://localhost:8081"),
```

Comment:
with `emptyMap` this test throws `Either schema.registry.url or keyAvroSchema is required to create Avro-based mapping`. Did we stop supporting legacy `jet.sql` schema (first in https://docs.hazelcast.com/hazelcast/5.3/sql/mapping-to-kafka#avro-messages)?

Reply:
I didn't enable auto-generated schemas for our (de)serializer. It is only supported for Confluent's (de)serializer, which are automatically chosen in the presence of `schema.registry.url`. However, if our schema registry will support `auto.register.schemas` mode, I need to enable it.

Reply:
this is a breaking change, see first example in the docs. `jet.sql` without schema registry was poorly usable, but could be used if you created equivalent schema on the other side or only used it for `sql -> kafka -> sql` flows.

Reply:
I enabled schema auto-generation for `HazelcastKafkaAvro[De]Serializer`.